packages/synapse-react-client/src/components/ReleaseCard/ReleaseCardUtils.ts

```diff
@@ -101,23 +151,27 @@ export const createButtonToExploreDataPathAndQueryString = (
   }
 
   const {
-    exploreDataSql,
-    exploreDataFacetColumnName,
-    sourceDataFacetValueColumnName,
+    sourceExploreDataSqlColumnName,
+    selectedFacetConfigs,
+    staticSelectedFacets,
   } = buttonToExploreDataConfig
 
-  const facetValue = sourceDataFacetValueColumnName
-    ? getValueFromData(schema, data, sourceDataFacetValueColumnName)
+  const exploreDataSql = sourceExploreDataSqlColumnName
+    ? getValueFromData(schema, data, sourceExploreDataSqlColumnName)
     : null
-  const hasSelectedFacet =
-    exploreDataSql && exploreDataFacetColumnName && facetValue
+  const allSelectedFacets = getAllSelectedFacets(
+    schema,
+    data,
+    selectedFacetConfigs,
+    staticSelectedFacets,
+  )
+  const hasSelectedFacets = exploreDataSql && allSelectedFacets.length > 0
```

Comment:
Lookup explore page SQL in the source table and allow user to pass facet values that are looked up in the source table (selectedFacetConfigs) or are statically defined (staticSelectedFacets)